legal-api/src/legal_api/models/filing.py

```diff
@@ -112,6 +112,10 @@ class Source(Enum):
                 'BEN': 'BCINC'
             }
         },
+        'conversion': {
+            'name': 'conversion',
+            'title': 'Conversion Ledger'
+        },
```

Comment:
Removed this thinking that it was not required and used only for retrieving fee codes used for passing to the payment api during the invoice creation process.  After testing out the latest code in the main branch, conversion filings are not being processed by the `entity-filer` queue worker process due to this piece of missing code.  Without this code, the conversion filing processing logic is never triggered. 

Note that no fee codes(`codes` property) have been added as the colin conversion filing should not trigger the create invoice logic nor should it require any payment.